src/harness/util.py

```diff
@@ -405,9 +405,9 @@ def filterChannelsByFrequencyRange(channels, freq_range):
   return [
```

Comment:
Can you please update the docstring to read "Returns channels which partially or fully overlap with 'freq_range'." ? Then the intention of this function is clearer.

Reply:
Kate answered my question. Thanks. Approved after updating the docstring.